.github/workflows/inductor-perf-test-nightly.yml

```diff
@@ -2,7 +2,7 @@ name: inductor-A100-perf-nightly
 
 on:
   schedule:
-    - cron: 45 1 * * *
+    - cron: 45 1,7,13,19 * * *
```

Comment:
Is 4 times a day too frequent? Maybe we can set to 2 and see if it gives sufficient amount of information that is needed. 
If the job is producing more information than we can consume, or if there are less amount of delta, e.g. two adjacent night runs where the commits may be identical or closely identical, there may not be a need to run 4 times. 2 or 3 may be a good balance?

Reply:
Sounds good.